SPECS/Cython/Cython.spec

```diff
@@ -1,79 +1,396 @@
-%{!?python2_sitelib: %define python2_sitelib %(python2 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
-Summary:       C extensions for Python
-Name:          Cython
-Version:       0.28.5
-Release:       7%{?dist}
-Group:         Development/Libraries
-License:       ASL 2.0
-URL:           https://cython.org
-#Source0:      https://github.com/cython/cython/archive/%{version}.tar.gz
-Source0:       %{name}-%{version}.tar.gz
-Patch0:        fix_abc_tests.patch
-Vendor:        Microsoft Corporation
-Distribution:  Mariner
-BuildRequires: python2-devel
-BuildRequires: python2-libs
-BuildRequires: python-xml
-Requires:      python2
-
-%description
-Cython is an optimising static compiler for both the Python programming language and the extended Cython programming language (based on Pyrex). It makes writing C extensions for Python as easy as Python itself.
+%{!?python3_sitelib: %define python3_sitelib %(python3 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
+%define python3_sitearch %(python3 -c "from distutils.sysconfig import get_python_lib; import sys; sys.stdout.write(get_python_lib(1))")
+
+%global upname cython
+
+Name:           Cython
+Version:        0.29.13
+Release:        6%{?dist}
+Summary:        Language for writing Python extension modules
+Vendor:         Microsoft
```

Comment:
Same comment, please use "Microsoft Corporation". Please also update other spec files from this PR, if they have the same issue.

Reply:
Done